src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -204,4 +206,17 @@ public ResponseEntity<CaseDetails> caseworkerSubmitEvent(
         }
     }
 
+    @GetMapping(path = "/{caseId}/userCaseRoles")
+    @Operation(summary = "Get user Roles for a case")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(responseCode = "400", description = "Not Found"),
+        @ApiResponse(responseCode = "401", description = "Not Authorized")})
+    public ResponseEntity<List<String>> getUserInfo(
+        @PathVariable("caseId") String caseId,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorization) {
+        List<String> roles = userInformationService.getUserCaseRoles(caseId, authorization);
+        return new ResponseEntity<>(roles, HttpStatus.OK);
```

Comment:
can we place the call in the ok response body?
```suggestion
        return new ResponseEntity.ok(userInformationService.getUserCaseRoles(caseId, authorization));
```